### ISSUE TITLE
[ERROR] Unable to retrieve local hardware address libnet ( ioctl: Permission denied ).

### ISSUE THREAD

**Issue by stanIee** [topjohnwu/Magisk#2282]
I'm reopening this issue since it's not a problem only with me. So we shall not close it until we have some more information on why this problem pops out and how to fix it. Like I said people with Magisk installed had this error with these apps: zAnti, dsploit.

**Comment by linuxandria** [comment]
These apps are as suggested rather old and may not be able to handle newer su solutions.

Anyhow as I said _not a magisk issue_, it's a problem with the apps. Seeing as no root apps I use have problems evidently magisk works fine. Magisk is also the only actively maintained root solution (excluding niche su solutions like phh superuser), so if it's designed to work with eg SuperSU...


Recommend closing. App issue, talk to the app developer.

**Comment by topjohnwu** [comment]
First, you didn't provide any useful information for me to debug anything
Second, as @linuxandria said, this is very likely an app issue, not a Magisk issue

Please don't open a new issue on this same topic unless you have insight on what caused the bug
